v0/destinations/facebook_pixel/transform.js

```diff
@@ -169,25 +123,30 @@ const handleProductListViewed = (message, categoryToContent) => {
   if (contentIds.length > 0) {
     contentType = "product";
   } else {
-    contentIds.push(message.properties.category || "");
+    //  for viewContent event content_ids and content arrays are not mandatory
+    if(message.properties?.category) {
```

Comment:
We added this validation here did not understand this part

Reply:
because all the fields under this are non-mandatory